src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('.dashboard');
+  const thead = table.querySelector('thead');
+  const tbody = table.querySelector('tbody');
+  const tfoot = table.querySelector('tfoot');
+
+  const cloneColumn = (section) => {
+    const rows = section.querySelectorAll('tr');
+
+    rows.forEach((row) => {
+      const cells = row.querySelectorAll('th, td');
+      const cloneCell = cells[1].cloneNode(true);
+
+      row.insertBefore(cloneCell, cells[4]);
+    });
+  };
+
+  cloneColumn(thead);
```

Comment:
Now there is no cloned column in the table:

![image](https://github.com/mate-academy/js_task_clone_table_column_DOM/assets/86923089/0366c3ee-a22e-4c8e-90ba-32183ffab8ae)

Make sure it works on the deployed variant
Consider fixing tests (ask for help in the chat if you need)